app/assets/javascripts/blacklight/blacklight.js

```diff
@@ -432,54 +432,68 @@ Blacklight.onLoad(function () {
   Blacklight.modal.setupModal();
 });
 
-(function ($) {
-  Blacklight.doSearchContextBehavior = function () {
-    if (typeof Blacklight.do_search_context_behavior == 'function') {
-      console.warn("do_search_context_behavior is deprecated. Use doSearchContextBehavior instead.");
-      return Blacklight.do_search_context_behavior();
-    }
+Blacklight.doSearchContextBehavior = function () {
```

Comment:
Note: this code is the compiled distribution. The source is in `app/javascript/`